typing/mode.ml

```diff
@@ -2170,13 +2193,40 @@ module Monadic = struct
     include C.Monadic
     include Lattice_Product (C.Monadic)
 
-    let print_axis ax ppf a =
-      let obj = proj_obj ax in
-      C.print obj ppf a
+    module Per_axis = struct
+      let print ax ppf a =
+        let obj = proj_obj ax in
+        C.print obj ppf a
 
-    let le_axis ax a b =
-      let obj = proj_obj ax in
-      C.le obj b a
+      let le ax a b =
+        let obj = proj_obj ax in
+        C.le obj b a
+
+      let join ax a b =
+        let obj = proj_obj ax in
+        C.meet obj a b
+
+      let meet ax a b =
+        let obj = proj_obj ax in
+        C.join obj a b
```

Comment:
maybe comment on why the joins and meets here are backwards? I got spooked initially